### PR TITLE
paas button

### PR DESCRIPTION
**Pull request by xi**

### CODE REVIEW

File: src/s1/s1/static/stylesheets/scss/components/_button_theme.scss [hunk 1]
```diff
@@ -64,44 +64,6 @@
...
-    &:disabled {
-        &:hover,
-        &:focus,
-        &:active {
-            background: $color-brand-two-normal;
-            color: $color-text-inverted;
-        }
-    }
-}
-
```
Comment (der-john) [inline src/s1/s1/static/stylesheets/scss/components/_button_theme.scss]: nice.

File: src/s1/s1/static/stylesheets/scss/components/_button_theme.scss [hunk 2]
```diff
@@ -17,6 +17,10 @@
             color: $color-brand-one-extrovert;
         }
     }
+
+    &.m-round {
+        min-width: 2em;
+    }
```
Comment (der-john) [inline src/s1/s1/static/stylesheets/scss/components/_button_theme.scss]: why is this in s1 and not in core?

**Comment by der-john** [comment]
looks nice & works.

File: src/s1/s1/static/stylesheets/scss/components/_button_theme.scss (hunk 2 above)
Comment (xi) [inline src/s1/s1/static/stylesheets/scss/components/_button_theme.scss]: In paas buttons are taller, so they also need to be wider in order to be round.

File: src/s1/s1/static/stylesheets/scss/components/_button_theme.scss (hunk 1 above)
Comment (der-john) [inline src/s1/s1/static/stylesheets/scss/components/_button_theme.scss]: @dwt told me this was actually intended UI behavior. I'm making a PR.

**Pull request merged**